src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
```

Comment:
Thanks @xpdavid. Was more concerned about the logic errors in if-else, moving it into those functions is a nice way of avoiding all that.